docs/source/pages/cli_usage/command_analyze.rst

```diff
@@ -1,4 +1,4 @@
-.. Copyright (c) 2023 - 2023, Oracle and/or its affiliates. All rights reserved.
+.. Copyright (c) 2023 - 2025, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hmmm, I don't think this file's copyright notice should be updated.

Reply:
I think I noticed that the copyright year was not updated even though the content is changes in previous PRs. I will remove it from my PR, but we should watch for these updates because we don't automatically change the headers in rst files.

Reply:
See commit [a684508](https://github.com/oracle/macaron/pull/1021/commits/a6845083b70a170af22416b1f8cce556e16ca5dc).